src/controllers/commentController.js

```diff
@@ -9,6 +11,15 @@ const CommentsController = {
       userId: req.userData.userId,
       requestId: req.params.requestId,
     });
+    const { lineManager, id } = await UserService.getUser({ id: req.request.UserId });
+    // CREATE NOTIFICATION FOR THE REQUESER OR MANAGER
+    await NotificationService.createNotification({
+      modelName: 'Comments',
+      modelId: newComment.get({ plain: true }).id,
```

Comment:
what about just calling `newComment.id`

Reply:
that object is a direct instance of `Comments Model`, I can't access `.id` directly!

Reply:
I've worked on it!